x-pack/plugins/apm/common/utils/term_query_client.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { isNil, isEmpty } from 'lodash';
+import { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
+
+interface TermQueryOpts {
+  queryEmptyString: boolean;
+}
+
+export function termQueryClient<T extends string>(
```

Comment:
Don't like it that much. 

As we use more and more term queries on the client side maybe we should allow importing the helpers on the client

`import { termQuery, termsQuery } from '@kbn/observability-plugin/server';`

any thoughts?

Reply:
I needed it [here](https://github.com/elastic/kibana/blob/283754629b90123d2a2caf06e831b87b833645b5/x-pack/plugins/apm/public/components/app/service_overview/service_overview_charts/latency_map/use_map_filters.ts#L21) too. I think it makes sense to move the utils from the observability plugin to `common`.